src/main/java/org/openjdk/jextract/impl/HeaderFileBuilder.java

```diff
@@ -223,15 +227,20 @@ void emitPointerTypedef(Declaration.Typedef typedefTree, String name) {
 
     void emitFirstHeaderPreamble(List<String> libraries) {
         incrAlign();
-        appendLines("static final SymbolLookup SYMBOL_LOOKUP;");
+        appendLines("""
+
+            static final SymbolLookup SYMBOL_LOOKUP;
+            """);
         appendLines("static {");
         for (String lib : libraries) {
             String quotedLibName = lib.replace("\\", "\\\\"); // double up slashes
             String method = quotedLibName.indexOf(File.separatorChar) != -1 ? "load" : "loadLibrary";
             appendIndentedLines(STR."System.\{method}(\"\{quotedLibName}\");");
         }
-        appendLines("""
-
+        if (!libraries.isEmpty()) {
+            appendBlankLine();
```

Comment:
Do we really need a blank line here? Aren't we inside a method body? (so this is code, extra space is not needed here?)

Reply:
Yeah, it could be omitted. The current code has the intent to add one so I kept that.